ztp/siteconfig-generator-kustomize-plugin/testSiteConfig/site2-standard-du.yaml

```diff
@@ -13,7 +13,7 @@ spec:
   sshPrivateKeySecretRef:
     name: "sshPrvKey"
   biosConfigRef:
-    filePath: "testSiteConfig/testHW.profile"
+    filePath: "../siteconfig-generator-kustomize-plugin/testSiteConfig/testHW.profile"
```

Comment:
No need to change paths. for kustomize, the visible path starts after siteconfig-generator-kustomize-plugin/